src/include/sof/trace/trace-verbose-private.h

```diff
@@ -0,0 +1,557 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2019 Intel Corporation. All rights reserved.
+ *
+ * Author: Artur Kloniecki <arturx.kloniecki@linux.intel.com>
+ */
+
+#ifndef __SOF_TRACE_TRACE_VERBOSE_PRIVATE_H__
+#define __SOF_TRACE_TRACE_VERBOSE_PRIVATE_H__
+
+#include <config.h>
+
+#if CONFIG_TRACE_VERBOSE_IRQ
+#define tracev_TRACE_CLASS_IRQ(...)		\
+	trace_event(__VA_ARGS__)
```

Comment:
Wait, so how does trace_event still get the class? It's required for `sof-logger` tool (it displays the class for each event).